scm/src/run_scm.py

```diff
@@ -490,10 +490,10 @@ def setup_rundir(self):
             if input_type == 1:
                 #open the case data file and read the surfaceForcing global attribute
                 case_data_dir = case_nml['case_config']['case_data_dir']
-                nc_fid = Dataset(os.path.join(SCM_ROOT, case_data_dir) + '/' + self._case + '_SCM_driver.nc' , 'r')
```

Comment:
I don't think we need these changes to lines 493-494? (Just the change to line 496)

Reply:
@dustinswales Thanks, I reverted the L493 change. I just tested and it is working fine (note I made that change awhile back due to some random issue reading the file). Thinking about it, I wonder if surface_forcing_moisture should also be considered here, since they can be set independently. I'm not sure how the SCM the SCM would react with them set differently without looking more at the code.

Reply:
@dustinswales Revisiting this reverted change. It looks like changes are needed here otherwise the fluxes are not read in correctly. case_nml['case_config']['case_data_dir'] is empty, so either it needs to be assigned the path somewhere above, OR remove L492 and modify L493 case_data_dir > self._case_data_dir. This only shows up when prescribing sfc_fluxes. While the case will appear to run normally with a fix, it will run the default suite and not the *_ps version since nothing is getting read correctly here.